### PR TITLE
Add a 'Start a new sequence' button

### LINKED ISSUE #496: Add a 'Start a new sequence' button
### Please describe the problem (or idea)
Add a `start a new sequence` button in the header which when clicked, opens a new page 📄  for editing a new picture. The output of this button could either be loaded on the same tab or be opened in a new tab. 
> What did you expect to see that you didn't?

screenshot 37

### Please show us where to look

https://sequencer.publiclab.org

### What's your PublicLab.org username?
oorjit_chowdhary
### Browser, version, and operating system
Chrome, Windows 10

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by Mridul97**
Fixes #496 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Hi, @Mridul97 is this almost the same as #581 by @aashna27 ? If so, perhaps you two could collab or review each others' solutions and submit a joint one? 

Perhaps it should open in a new window in case people lose their work, too?

**Comment by Mridul97** [comment]
Oh! Yes it seems quite similar, both are opening a new page with no steps but in this case we are opening it in a new tab. So people would not lose their work. 

What would be better to click on the button "Start a new sequence" to open a new page or clicking on the header or both, depending on that maybe we can merge one or both?

I think having a seperate button makes it more clear to the user. Thanks!

**Comment by aashna27** [comment]
@Mridul but why add buttons unnecessarily when it can be done with the header, also most websites have this thing that you click on the logo or the name and it takes you to the home page. What say?

**Comment by jywarren** [comment]
I think both are different ways of thinking of similar functionality - one
reason to have a button might be that people may not know how to "clear"
the sequence and start a new one, and it does seem like in a web app of
this complexity, "starting a new sequence" may not seem obviously the way.
I'm also concerned about people losing their work, so what do you think
about having both a minimal button like [+ New sequence] AND the header do
this, but both open a new window?

Thanks for thinking through this carefully!

**Comment by HarshKhandeparkar** [comment]
If the header restarts the sequence then there is a high chance that the user may click the header by mistake and loose the work. Instead i think the header should open a new page with a new sequence and the foooter should have a button which can clear the current sequence but with a conformation.

**Comment by aashna27** [comment]
> If the header restarts the sequence then there is a high chance that the user may click the header by mistake and loose the work. Instead i think the header should open a new page with a new sequence and the foooter should have a button which can clear the current sequence but with a conformation.

yess seems perfect !!

**Comment by jywarren** [comment]
super!

**Comment by Mridul97** [comment]
@jywarren Please have a look!

reset

**Comment by HarshKhandeparkar** [comment]
@Mridul97 I think the button should confirm if the user wants to reset the sequence. The user may click on that button instead of clicking on download png and may loose progress.

**Comment by HarshKhandeparkar** [comment]
Cool that will be awesome 😊

On 07-Jan-2019 11:59 PM, "Mridul97" <notifications@github.com> wrote:

@HarshKhandeparkar <https://github.com/HarshKhandeparkar> Nice! I will add
a confirmation option. Thanks!

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/publiclab/image-sequencer/pull/589#issuecomment-452033605>,
or mute the thread

**Comment by HarshKhandeparkar** [comment]
@Mridul97 you can shorten the if statement to:
```
if(r) location.reload();
```

**Comment by Mridul97** [comment]
Done! Here is the new output : 

reset2

File: examples/index.html [hunk 1]
```diff
@@ -171,6 +171,7 @@ <h4 class="modal-title">Your gif is ready</h4>
         <div class="panel-body">
           <div style="text-align:center;">
             <button class="btn btn-success btn-lg" id="download-btn" name="download">Download PNG</button>
+             <input type="button" id="resetButton" class="btn btn-success btn-lg" value="Reset Sequence"></a>
```
Comment (HarshKhandeparkar) [inline examples/index.html]: There is still a `</a>` remaining here @Mridul97.

**Comment by HarshKhandeparkar** [comment]
Looks awesome @Mridul97.

**Comment by jywarren** [comment]
Awesome! Can we have it be a white button (btn-default, I think?) And say "clear all steps"? Since restart might sound like just rerunning everything. 

Thanks!!

**Comment by Mridul97** [comment]
@jywarren I have made the changes. Please have a look!

**Pull request merged**